accounts/abi/bind/backends/simulated.go

```diff
@@ -650,6 +650,14 @@ func (m callmsg) GasPrice() *big.Int   { return m.CallMsg.GasPrice }
 func (m callmsg) Gas() uint64          { return m.CallMsg.Gas }
 func (m callmsg) Value() *big.Int      { return m.CallMsg.Value }
 func (m callmsg) Data() []byte         { return m.CallMsg.Data }
+func (m callmsg) IsAA() bool           { return m.CallMsg.IsAA }
+func (m callmsg) Sponsor() common.Address {
```

Comment:
Would Payee() be a better term?

Reply:
Also, did you run the tests? I imagine this breaks some other tests/code out there?

Reply:
> Would Payee() be a better term?

Could be. Probably not important enough to change for MVP though?

Reply:
> Also, did you run the tests? I imagine this breaks some other tests/code out there?

AFAICT it does not break any tests, it just adds functions to the `callmsg` struct that are never used for the existing tests.